### ISSUE TITLE
CSS is output as "main.css", old esbuild w/limited plugins, and esbuild config not watched

### ISSUE THREAD

**Issue by pjeby** [zephraph/obsidian-tools#50]
Running the build or dev commands generates main.css in the target directory, rather than styles.css.  This appears to be esbuild's doing.  I was able to work around the problem by installing a local, up-to-date copy of esbuild, deleting the bundled esbuild from obsidian-plugin-cli, and using the following esbuild.config.js:

```js
const fs = require('fs');
let plugin = {
    name: "rename-styles",
    setup(build) {
        build.onEnd(() => {
            const {outfile} = build.initialOptions;
            const outcss = outfile.replace(/\.js$/, ".css");
            const fixcss = outfile.replace(/main\.js$/, "styles.css");
            if (fs.existsSync(outcss)) {
                console.log("Renaming", outcss, "to", fixcss);
                fs.renameSync(outcss, fixcss);
            }
        });
    }
}
module.exports = {
    plugins: [plugin],
}
```

Perhaps something like this plugin should be added to the build process by default?  (Also, bumping the esbuild version seems like a good idea, since the bundled version doesn't provide an onEnd hook.)

Last, but not least, it appears that the dev command does not watch esbuild.config.js, so you have to kill and restart it for config changes to take effect.

**Comment by zephraph** [comment]
Thanks for the feedback! Yeah, I'll tackle this.

**Comment by zephraph** [comment]
Okay the esbuild version bump should be going out.

**Comment by pjeby** [comment]
Sure.  By the way, rather than having a special option for a style sheet, shouldn't the plugin just import any css or scss files it needs?  There's no need to specify the file externally then.

**Comment by zephraph** [comment]
Yep, that is indeed the ideal. I didn't recommend that b/c of this message from esbuild's docs.

![image](https://user-images.githubusercontent.com/3087225/120910626-5e7e0d00-c64e-11eb-8f2b-1967ef472db5.png)

That referenced https://github.com/evanw/esbuild/issues/465 which seems to be fixed now. I'll update accordingly.
